### ISSUE TITLE
training summary should include also the current training loss

### ISSUE THREAD

**Issue by yardencsGitHub** [NickleDave/vak#182]
The current training log saves an 'accuracy' and a 'loss' metric for every validation point.
These are calculated on the validation set and can be easily parsed (e.g. in attached notebook)
Is it possible to add the current loss on the last training step (or averaged on a few)?
It will allow tracking the convergence.
[SummarizeTraining.zip](https://github.com/NickleDave/vak/files/4523297/SummarizeTraining.zip)

**Comment by NickleDave** [comment]
The training loss does get added to the tf.events file on every step, along with the val loss and acc:
https://github.com/NickleDave/vak/blob/84b3d228b39e9e3787ba32a8d2697b8d71857014/src/vak/engine/model.py#L120

I can add a util function to parse the tf.events file later if you need it.
Something like this
https://gist.github.com/ptschandl/ef67bbaa93ec67aba2cab0a7af47700b

The purpose of the the text log is to track events that happen, not to track the training history. Don't make things harder on yourself writing a function to parse that text file.

**Comment by yardencsGitHub** [comment]
Thanks. No need to spend your time on this. Closing.
